libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/hwdef.dat

```diff
@@ -43,6 +43,16 @@ PC9  TIM8_CH4 TIM8 PWM(8) GPIO(57)
 PB11 VTX_PWR OUTPUT HIGH GPIO(81)
 define RELAY2_PIN_DEFAULT 81
 
+# second battery sensor on second ESC connector
+PA1 BATT2_VOLTAGE_SENS ADC1 SCALE(1)
```

Comment:
this would be nice: 
```
PA1 BATT2_VOLTAGE_SENS ADC1 SCALE(1) PINNUM(13)
```

Reply:
I could not see how to do this - maybe a different PR

Reply:
yep, definately separate PR